### DISCUSSION TITLE
Change the opacity of an Instance on hover

### DISCUSSION THREAD

**Discussion by shurik** [pmndrs/drei#710] [Q&A]
Hello,

Could anyone point at a resource on how to change the opacity of an `<Instance/>`. I see how I can change color from the examples but opacity escapes me.

Really appreciate any insight

**Comment by drcmda** [comment]
instances do not have opacity. an instance is just one geometry and one material, that's virtually being applied to multiple objects at the shader level. you can of course use your own shader and send down the opacity as an buffer attribute, that would allow you to make opacity instanc...iable?
